api/queries/community/metaData.js

```diff
@@ -1,4 +1,4 @@
-// @flow
+// TODO: Flow type again
```

Comment:
Was the flow typing in this file super broken or something? Ideally we don't regress flow coverage, but I understand if it was a nightmare.